vendor/bat-native-ads/src/bat/ads/internal/catalog/catalog_unittest.cc

```diff
@@ -459,11 +457,10 @@ TEST_F(BatAdsCatalogTest, ParseCatalog) {
 
 TEST_F(BatAdsCatalogTest, ParseEmptyCatalog) {
   // Arrange
-  const absl::optional<std::string> opt_value =
-      ReadFileFromTestPathToString(kEmptyCatalog);
-  ASSERT_TRUE(opt_value.has_value());
-
-  const std::string json = opt_value.value();
+  const absl::optional<std::string> json_optional =
+      ReadFileFromTestPathAndParseTagsToString(kEmptyCatalog);
+  ASSERT_TRUE(json_optional.has_value());
+  const std::string json = json_optional.value();
```

Comment:
optional<>::value() returns a reference. So this could be: `const std::string& json = json_optional.value();`
this change avoids extra string copy.

Reply:
Fixed, thanks

Reply:
Fixed